tests/measure-perf.bats

```diff
@@ -162,6 +162,37 @@ function get_perf_opts() {
   [ "$npng" -eq 0 ] # no detailed charts because of --skip-perf
 }
 
+@test "measure perf --skip-perf --use-vdenc raw h264" {
+  tmp=`mktemp -p $_TMP -d -t demo-XXXX`
+  run vdenc_support H264High
```

Comment:
Wait... We will run all encoders here, right? Well, this won't do in this case since you just checks for AVC encoder capability... I think we need to break this into 2 tests: for AVC and HEVC encoders, i.e.:
```
@test "measure perf --skip-perf --use-vdenc --codec AVC raw h264" {
  tmp=`mktemp -p $_TMP -d -t demo-XXXX`
  run vdenc_support H264High
  if [ $status -eq 0 ]; then skip; fi  
  run docker_run_opts "$(get_perf_opts $tmp)" /bin/bash -c " \
    $(get_test_body "$rawh264" "measure perf --skip-perf --use-vdenc --codec AVC /tmp/WAR.h264")"
  ...
}

@test "measure perf --skip-perf --use-vdenc --codec HEVC raw h264" {
  tmp=`mktemp -p $_TMP -d -t demo-XXXX`
  run vdenc_support HEVCMain
  if [ $status -eq 0 ]; then skip; fi  
  run docker_run_opts "$(get_perf_opts $tmp)" /bin/bash -c " \
    $(get_test_body "$rawh264" "measure perf --skip-perf --use-vdenc --codec HEVC /tmp/WAR.h264")"
  ...
}
```

Reply:
hi dima, there are two tests.. 

@test "measure perf --skip-perf --use-vdenc --codec AVC raw h264" {
  tmp=`mktemp -p $_TMP -d -t demo-XXXX`
  run vdenc_support H264High

@test "measure perf --skip-perf --use-vdenc raw h265" {
  tmp=`mktemp -p $_TMP -d -t demo-XXXX`
  run vdenc_support HEVCMain

Reply:
each of these tests runs both avc and hevc encoders... but you check only one of them...